test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +46,33 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        // empty List (size 0)
+        assertIsNotNull();
+
+        // only one object
+        assertIsNull(new Object[] {null});
+        assertIsNotNull("a");
+
+        // all null objects
+        assertIsNull(new Object[] {null, null, null});
+
+        // all assigned objects
+        assertIsNotNull(1,2,3);
+        assertIsNotNull("a", 10, new Object());
+        assertIsNotNull(new LinkedList<>(), new HashSet<>());
+
+        // some null objects
+        assertIsNull(null, 1, null);
+    }
```

Comment:
Good job on attempting this LO!

It's great that you added comments to different sections of your test code to aid in readability and debugging. Good to see that you are also testing a wide combination of inputs!

Keep it up!